### DISCUSSION TITLE
Welcome to TSS.MSR Discussions!

### DISCUSSION THREAD

**Discussion by DMattoon** [microsoft/TSS.MSR#85] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mwiseman32** [comment]
Dennis,
What is the status of the PCPTool. There have been no recent updates. EnumerateKeys doesn't support ECC keys and it exits with an error on the first one it finds. It also doesn't compile by just downloading as there are several redefinitions in the header files.
